src/library/components/page-header.js

```diff
@@ -149,106 +132,103 @@ var PageHeader = Component({
     if (!this.state.isStudent) {
       if (this.state.theme === 'ngss-assessment') {
         headerItems.push(
-          li({className: 'portal-pages-main-nav-item' +
-              ' portal-pages-main-nav-collections' +
-              (this.props.isCollections ? ' current-menu-item' : '')},
-          a({href: '/ngsa-collections', className: 'portal-pages-main-nav-item__link', title: 'View Assessment Tasks'},
-            'Assessment Tasks'
-          )
-          ))
+          <li className={'portal-pages-main-nav-item' + ' portal-pages-main-nav-collections' + (this.props.isCollections ? ' current-menu-item' : '')}>
+            <a href={'/ngsa-collections'} className={'portal-pages-main-nav-item__link'} title={'View Assessment Tasks'}>{'Assessment Tasks'}</a>
```

Comment:
The title attribute could be simplified to `title='View Assessment Tasks'`. This could also be applied to the href and className attribute.

Reply:
It seems like the convent is to use double quotes for attributes, but perhaps our linting system will change them automatically to single quotes.